pom.xml

```diff
@@ -353,6 +353,45 @@
       </build>
     </profile>
 
+    <profile>
+      <id>errorprone</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <!-- Turn on Error Prone -->
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <version>3.3</version>
```

Comment:
This is just copy/paste from their example. I think I can remove the version here if it is inherited from elsewhere.